data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,160 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"Alright, I'll show you to my ship."`
+					goto accept
+				`	"Why weren't they built on Second Cerulean in the first place?"`
+				`	"Sorry, I'm not headed that way."`
+					decline
+			`	"No facilities for such construction, Second Cerulean has," he explains. "Remain mostly untouched, the planet must. Build more than simple research stations on it, we cannot. But in need of sturdier subs, we are, so held a competition to determine what company was to build them, my group did. In my opinion, honestly, not needed such a process was. Obvious that House Mallob would win, it was, unrivaled in hull integrity, they are."`
+			choice
+				`	"Fair enough. I'll show you to my ship."`
+				`	"Well, I wish you luck, but I'm not headed that way, sorry."`
+					decline
+			label accept
+			`	"Very good. Thank you, Captain. Called Apri'ie, I am, member of the Pale Waters research center. A wonderful trip, may we have." You bring him to your ship, and help him settle what little luggage he has in his room.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Apri'ie hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		"coalition jobs" ++
+		payment 90000
+		conversation
+			`Apri'ie has you to land by one of House Mallob's offices, which looks like a mix of a skyscraper and a warehouse, and hands you <payment>.`
+			`	"Thank you for the assistance, Captain <last>. Meet with House Mallob's executives, I must, but if to continue helping, you wish, meet you in the spaceport, I will. The proper transportation of the submarines, the next step is, so if interested, you are, free up space for them, you must." He pauses, as if counting the total weight. "About 60 tons, they should take up. At any rate, talk more in detail in the spaceport, we may." He picks up his belongings and heads for the office, disappearing into the building.`
+
+mission "Coalition: Submarines 2"
+	name "Pressurized Stress Test"
+	description "Bring Apri'ie and the <cargo> to <destination>, where the submarines will be subjected to some final high-pressure stress tests."
+	passengers 1
+	cargo "unmanned submarines" 60
+	blocked "You need <capacity> to take the next mission. Return here when you have the required space free."
+	to offer
+		has "Coalition: Submarines 1: done"
+	source "Market of Gupta"
+	destination "Inmost Blue"
+	on offer
+		conversation
+			`After asking around for a member of House Mallob, you're informed that Apri'ie hasn't left his meeting yet, and so you take a seat in the spaceport, occasionally getting up to check back at the entrance of the office he entered. After some hours, you finally see him coming out, accompanied by a Saryd speaking on a communication device as they glance at a data pad.`
```

Comment:
Apologies if I'm misinterpreting, but I don't think it would make sense for some random member of House Mallob pulled off the street to know both Apri'ie and his location. I'm assuming that House Mallob is fairly populous, and that it has multiple business deals going on at the same time as the mission string. If this is true, then the likelihood of the player being able to find a member of House Mallob that knows of this specific business deal with this specific Kimek is quite low. Wouldn't it make more sense for the player to just call Apri'ie instead?

Reply:
I dunno, it wouldn't be hard to find someone who at least knows who to direct you to, even if you have to speak with an intern first. At that point they just need to check what schedule or whatever may be provided in their systems, and inform the player, who would warrant the information, being Apri'ie's transport.

Reply:
Hmm, okay. Perhaps you could add something about the player being directed to someone who knows about Apri'ie, because as it stands, it sounds like the player just found someone in House Mallob who happened to know exactly who Apri'ie is and when his meeting is ending.

Reply:
Is it better now?

Reply:
Yeah, looks good.